components/kyma-operator/pkg/kymahelm/kyma-helm-client.go

```diff
@@ -71,6 +75,34 @@ func (hc *Client) ReleaseStatus(rname string) (string, error) {
 	return strings.Replace(statusStr, `\n`, "\n", -1), nil
 }
 
+//IsReleaseDeletable returns true for release that can be deleted
+func (hc *Client) IsReleaseDeletable(rname string) (bool, error) {
+	isDeletable := false
+	maxAttempts := 3
+
+	err := retry.Do(
+		func() error {
+			statusRes, err := hc.helm.ReleaseStatus(rname)
+			if err != nil {
+				if strings.Contains(err.Error(), errors.ErrReleaseNotFound(rname).Error()) {
+					isDeletable = false
+					return nil
+				}
+				return err
+			}
+			isDeletable = statusRes.Info.Status.Code != release.Status_DEPLOYED
+			return nil
+		},
+		retry.Attempts(uint(maxAttempts)),
+		retry.DelayType(func(attempt uint, config *retry.Config) time.Duration {
+			log.Printf("Retry number %d on getting release status.\n", attempt+1)
+			return time.Duration(3) * time.Second
```

Comment:
```suggestion
			return time.Duration(fixedDelay) * time.Second
```
Let's avoid "magic" constants - it should be named properly. Declare `fixedDelay` next to maxAttempts.